documentation/general/analyzer-redirecting.md

```diff
@@ -0,0 +1,129 @@
+# Redirecting analyzers in SDK to VS
+
+We will redirect analyzers from the SDK to ones deployed in the VS to avoid the [torn SDK][torn-sdk] issue.
+Only major versions will be redirected because different major versions of the same analyzer are not compatible.
+So this applies to a situation like:
+- Having an analyzer in SDK 9.0.1 referencing Roslyn 4.12. That gets deployed to VS 17.12.
+- Having an analyzer in SDK 9.0.7 referencing Roslyn 4.13.
+- User loads project with SDK 9.0.7 in VS 17.12.
+  Previously the analyzers would fail to load as they reference newer Roslyn version (4.13) than what is part of VS (4.12).
+  Now we will redirect the analyzers to those from SDK 9.0.1 that are deployed as part of VS and reference the matching Roslyn (4.12).
+
+Loading analyzers with older major version should not be a problem because they must reference an older version of Roslyn.
+
+Loading analyzers with newer major version in an old VS will result in an error like:
```

Comment:
May want to specify that this is the existing behavior. Basically the proposal isn't causing this to happen.